### PR TITLE
Add translation of Tutorial file.

### PR DESCRIPTION
**Pull request by Lrv-dev**

### CODE REVIEW

File: docs/guide/tutorial-en-us.md [hunk 1]
```diff
@@ -1,42 +1,345 @@
...
+```
+
+Structure
+---------
+
+A great point to facilitate the learning, is its very well organized syntax of the language. With a quick view separated in rows and columns, where these have a fixed size of parameters, even when not used.
+
+A program line is consequently an instruction, which has three mandatory parameters. They are separated in the respectively order of **register**, **address**, **constant**.
+
+![structure_en><](https://3bc-lang.org/images/structure-pt-br.png)
```
Comment (RodrigoDornelles) [inline docs/guide/tutorial-en-us.md:61]: change to `structure_en ><`

Comment (RodrigoDornelles) [reply docs/guide/tutorial-en-us.md:61]: upload img if possible!

**Review by RodrigoDornelles — CHANGES_REQUESTED** [review]
Image

File: docs/guide/tutorial-en-us.md (hunk 1 above)
Comment (Lrv-dev) [reply docs/guide/tutorial-en-us.md]: ok

**Review by RodrigoDornelles — APPROVED** [review]

**Pull request merged**